scripts/add_currency_notes.py

```
@@ -0,0 +1,194 @@
+    "XOF": "West African CFA franc, shared by eight West African nations.",
+    "XPF": "CFP franc, used in French Polynesia, New Caledonia, and Wallis and Futuna.",
+    "YER": "Yemeni rial, Yemen has been in civil conflict since 2015.",
+    "ZAR": "South African rand, the most traded African currency globally.",
+    "ZMW": "Zambian kwacha, Zambia is one of Africa's largest copper producers.",
+    "ZWL": "Zimbabwean dollar, Zimbabwe experienced one of history's worst hyperinflations.",
+}
+
+h = {"xc-token": API_KEY, "Content-Type": "application/json"}
+
+# Fetch all records (id + Code + Notes)
+resp = requests.get(
```

Ruff (0.15.12)

[error] Probable use of `requests` call without timeout

(S113)